rosa_architecture/rosa-understanding.adoc

```diff
@@ -5,7 +5,59 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
+Learn about {product-title} (ROSA), interacting with ROSA using {cluster-manager-first} and command-line interface (CLI) tools, consumption experience, and integration with Amazon Web Services (AWS) services.
 
-include::modules/rosa-understanding.adoc[leveloffset=+1]
-include::modules/rosa-using-sts.adoc[leveloffset=+2]
+== About ROSA
+
+ROSA is a fully-managed turnkey application platform that allows you to focus on delivering value to your customers by building and deploying applications. Red Hat and AWS Site reliability engineering (SRE) experts manage the underlying platform so you do not have to worry about the complexity of infrastructure management. ROSA provides seamless integration with a wide range of AWS compute, database, analytics, machine learning, networking, mobile, and other services to further accelerate the building and delivering of differentiating experiences to your customers.
+
+You subscribe to the service directly from your AWS account.
+
+After the clusters are created, you can operate your clusters with the OpenShift web console or through {cluster-manager-first}. The ROSA service also uses OpenShift APIs and command-line interface (CLI) tools. These tools provide a standardized OpenShift experience to use your existing skills and tools knowledge.
+
+You receive OpenShift updates with new feature releases and a shared, common source for alignment with OpenShift Container Platform. ROSA supports the same versions of OpenShift as Red Hat OpenShift Dedicated and OpenShift Container Platform to achieve version consistency.
+
+== Credential modes
+
+There are two supported credential modes for ROSA clusters. One uses the AWS Secure Token Service (STS), which is recommended, and the other uses Identity Access Management (IAM) roles.
+
+=== AWS STS
+
+AWS STS is a global web service that provides short-term credentials for IAM or federated users. AWS STS is the recommended credential mode for ROSA clusters. You can use AWS STS with ROSA to allocate temporary, limited-privilege credentials for component-specific IAM roles. The service enables cluster components to make AWS API calls using secure cloud resource management practices.
+
+You can use the `rosa` CLI to create the IAM role, policy, and identity provider resources that are required for ROSA clusters that use STS.
+
+AWS STS aligns with principles of least privilege and secure practices in cloud service resource management. The `rosa` CLI manages the STS credentials that are assigned for unique tasks and takes action upon AWS resources as part of OpenShift functionality. One limitation of using STS is that roles must be created for each ROSA cluster.
+
+A listing of the account-wide and per-cluster roles is provided in xref:../rosa_architecture/rosa-sts-about-iam-resources.adoc#rosa-sts-about-iam-resources[About IAM resources for ROSA clusters that use STS].
+
+=== AWS without STS
```

Comment:
Header should be "ROSA without STS (not recommended and will be removed as an option in a future version)"